lib/vanagon/platform/windows.rb

```diff
@@ -34,6 +34,143 @@ def package_name(project)
         "#{project.name}-#{project.version}-#{project.release}-#{@architecture}.msi"
       end
 
+      def add_repository(definition)
+        definition = URI.parse(definition)
```

Comment:
If the repository to be added will always be a URL, "definition" is an undescriptive variable name. Consider something more descriptive like "url", "repo_url", "a_url_to_be_named_later", "there_are_many_urls_like_this_but_this_url_is_mine", etc.